### PR TITLE
Provide KEDA 2.0 release blog post

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

File: content/blog/keda-2.0-release.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+Learn more about how to migrate by using our [migration guide](https://keda.sh/docs/2.0/migration/)!
+
+> **⚠ Running KEDA 1.x & 2.0 side-by-side is not supported.**
+>
+> KEDA comes with a metrics server and Kubernetes only allows you to run one of them in a cluster.
+>
+>*Learn more about how KEDA is architected in [our docs](http://keda.sh/docs/latest/concepts/#architecture).*
+
+## KEDA 💘 Community
```
Comment (tomkerkhove) [inline content/blog/keda-2.0-release.md:106]: @jeffhollan This bit was added after your review, you might want to take a look at it.

File: content/blog/keda-2.0-release.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Getting started
+
+There are many ways to get started with KEDA:
+
+- Install with [OperatorHub.io](https://operatorhub.io/operator/keda)
+
+- Install with Helm:
+
+>$ helm repo add kedacore https://kedacore.github.io/charts
```
Comment (zroubalik) [inline content/blog/keda-2.0-release.md:29]: nit: might look better to squash those 3 commands into on block

Comment (tomkerkhove) [reply content/blog/keda-2.0-release.md:29]: Done

Comment (tomkerkhove) [reply content/blog/keda-2.0-release.md:29]: Done

**Pull request merged**